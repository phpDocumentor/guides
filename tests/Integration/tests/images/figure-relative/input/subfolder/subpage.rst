=======
Subpage
=======

Lorem Ipsum Dolor.

An image with absolute paths
============================

..  figure:: /images/hero2-illustration.svg
    :width: 400
    :alt: Alternative text

    Description


An image with relative paths
============================

..  figure:: ../images/hero2-illustration.svg
    :width: 400
    :alt: Alternative text

    Description

..  toctree::
    :hidden:
    :glob:

    subfolder/*
