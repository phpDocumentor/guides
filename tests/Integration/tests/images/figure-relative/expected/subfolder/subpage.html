<!-- content start -->
    <div class="section" id="subpage">
            <h1>Subpage</h1>

            <p>Lorem Ipsum Dolor.</p>
            <div class="section" id="an-image-with-absolute-paths">
            <h2>An image with absolute paths</h2>

            <figure>
    <img
    src="../images/hero2-illustration.svg"
    width="400"            alt="Alternative text"        />



                    <figcaption><p>Description</p>
</figcaption>
            </figure>

    </div>

            <div class="section" id="an-image-with-relative-paths">
            <h2>An image with relative paths</h2>

            <figure>
    <img
    src="../images/hero2-illustration.svg"
    width="400"            alt="Alternative text"        />



                    <figcaption><p>Description</p>
</figcaption>
            </figure>

    </div>

    </div>

<!-- content end -->
